.jscsrc-tools

```diff
@@ -1,10 +1,10 @@
 {
 	"preset": "jquery",
 	"preset": "google",
-	"validateIndentation": "\t",
+	"validateIndentation": 2,
```

Comment:
This is more likely a problem with your editor than the code. If it bothers you that much split into a different PR

Reply:
Yeah, I know Atom does soft tabs. spaces are just safer in general though

Reply:
I changed it back in a new commit rather than cherry-picking